docker-images/kafka/Dockerfile

```diff
@@ -5,7 +5,9 @@ ARG KAFKA_SHA512
 ARG KAFKA_VERSION
 ARG strimzi_version
 
-RUN yum -y install java-${JAVA_VERSION}-openjdk-headless gettext nmap-ncat openssl && yum clean all -y
+RUN yum -y --setopt=tsflag=nodocs update \
```

Comment:
Isn't it `tsflags` not `tsflag`?

Reply:
I guess you are right. It seems to be anyway set by default, so I removed it completely.